### PR TITLE
Add content-loader tool

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Add `content-loader` tool. This tool is to enable only fetching content for website, like a documentation from root repository, or (in the future) issues for roadmap section.

**Related issue(s)**
- https://github.com/kyma-project/website/issues/196

### CODE REVIEW

File: tools/content-loader/src/documentation/index.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    console.log(
+      `Generating documentation versions file to ${outputDocsVersion}`,
+    );
+    generateDocsVersions({
+      releases: newestReleases,
+      prereleases: filteredPrereleases,
+      branches,
+      output: outputDocsVersion,
+    });
+  } else {
```
Comment (pkosiec) [inline tools/content-loader/src/documentation/index.js]: Invert the condition and return

File: tools/content-loader/Makefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+TAG = $(DOCKER_TAG)
+
+ci-pr: resolve build-image push-image
+ci-master: resolve build-image push-image
+ci-release: resolve build-image push-image
+
+resolve:
+	npm install --no-optional
+	
+validate:
```
Comment (michal-hudy) [inline tools/content-loader/Makefile]: It is not used

File: tools/content-loader/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+| Name                          | Default value     | Description                                                 |
+| ----------------------------- | ----------------- | ----------------------------------------------------------- |
+| APP_TOKEN                     | `null`            | The GitHub API OAuth token.                                 |
+| APP_ORGANIZATION              | `kyma-project`    | The GitHub organization that owns a given repository.       |
+| APP_REPOSITORY                | `kyma`            | The repository with documentation.                          |
+| APP_DOCS_COMMIT               | `null`            | The commit from the `master` branch with new documentation. |
+| APP_DOCS_OUTPUT               | `out`             | The path for storing the documentation results.             |
+| APP_DOCS_VERSIONS_CONFIG_FILE | `versions.json`   | The path to the website documentation configuration.        |
+| APP_DOCS_TEMP_DIR             | `tempDir`         | The path for storing temporary data.                        |
+
```
Comment (kazydek) [inline tools/content-loader/README.md]: Could you put the parameter names in bold and remove periods from their descriptions? (they are not full sentences).

**Pull request merged**